server/schedulers/hot_region.go

```diff
@@ -804,13 +804,13 @@ func (bs *balanceSolver) calcProgressiveRank() {
 		greatDecRatio, minorDecRatio := bs.sche.conf.GetGreatDecRatio(), bs.sche.conf.GetMinorGreatDecRatio()
 		switch {
 		case byteHot && byteDecRatio <= greatDecRatio && keyHot && keyDecRatio <= greatDecRatio:
-			// Both byte rate and key rate are balanced, the best choice.
+			// If use the case, both byte rate and key rate will be more balanced, the best choice.
 			rank = -3
 		case byteDecRatio <= minorDecRatio && keyHot && keyDecRatio <= greatDecRatio:
-			// Byte rate is not worsened, key rate is balanced.
+			// If use the case, byte rate will be not worsened, key rate will be more balanced.
```

Comment:
```suggestion
			// If belong to the case, byte rate will be not worsened, key rate will be more balanced.
```
Or simply no need to change